CppCoreGuidelines.md

```diff
@@ -17528,7 +17528,19 @@ Because that's the best we can do without direct concept support.
 
 ##### Example
 
-    enable_if<???>
+    template <typename T>
+    enable_if_t<is_integral_v<T>>
+    f(T v)
+    {
+        // ...
+    }
+
+    // Equivalent to:
+    template <Integral T>
+    void f(T v)
```

Comment:
I like that it uses a real [C++20 concept](https://en.cppreference.com/w/cpp/concepts/Integral). but let's hear from other editors on whether the examples should stick to `template<Integral T> void f(T v)` or `void f(Integral autp v)`.

Reply:
I'm thinking about changing the example to use `void sort(Sortable&);` and a link to T.13.

Reply:
Never mind, there isn't a `std::is_sortable` trait. I would like to keep the `enable_if` example as short as possible.